themes/theme-gmd/pages/index.jsx

```diff
@@ -43,54 +44,57 @@ import * as routes from './routes';
 import themeApi from '../themeApi';
 
 const devFontsUrl = 'https://fonts.googleapis.com/css?family=Roboto:400,400i,500,500i,700,900';
+const themeConfigParse = new ConfigParser(themeConfig).parse();
```

Comment:
`themeConfigParse` should be called `themeConfigParsed`

Reply:
Done